### PR TITLE
Added Product listing page

### PR DESCRIPTION
**Pull request by ayush65**
Added ProductListing page

### CODE REVIEW

File: src/SideBar/SideBar.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <h1 className="sidebar-margin">Category</h1>
+        <ul className="sidebar-components-margin">
+        <li>
+        <input type="checkbox" className="sidebar-price-ranging price-range-below-100" name="price-ranging" value="price-range"/>
+        <label >Fruits And Vegetables</label>
+        </li>
+
+        <li>
+        <input type="checkbox" className="sidebar-price-ranging price-range-btw-100-500" name="price-ranging" value="price-range"/>
+        <label >Dairy And Egg</label>
```
Comment (NtshVrm) [inline src/SideBar/SideBar.js:75]: you can try making the category and filters into a components because code is repeating

**Review by NtshVrm — APPROVED** [review]
Other than the suggested change, everything looks good👌

File: src/SideBar/SideBar.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        </ul>
+
+
+
+
+        <h1 className="sidebar-margin">Category</h1>
+        <ul className="sidebar-components-margin">
+        <li>
+        <input type="checkbox" className="sidebar-price-ranging price-range-below-100" name="price-ranging" value="price-range"/>
+        <label >Fruits And Vegetables</label>
```
Comment (shubhambishnoi) [inline src/SideBar/SideBar.js:70]: try to use htmlFor attributes for input labels

**Review by shubhambishnoi — APPROVED** [review]
everything else looks good, good clean code

File: src/SideBar/SideBar.js (hunk 2 above)
Comment (ayush65) [reply src/SideBar/SideBar.js:70]: I HAVE USED IT WAS SHOWING WARNING (NETLIFY )FOR THAT SO I HAVE REMOVED IT

**Pull request merged**